woocommerce/storefront-product-pagination
# Disable extension if Storefront > 2.3.0

In Storefront 2.3.0 we decided to merge this extension into core. This PR disables the extension if the Storefront version is greater or equal to 2.3.0.

**To test:**

Install Storefront and this extension and test with and without this PR.

Without this PR, you'll see that the code from this extension is still being loaded. The default behaviour in 2.3.0 is to disable its own implementation if it finds that this extension is enabled.

See: https://github.com/woocommerce/storefront/blob/master/inc/woocommerce/storefront-woocommerce-template-functions.php#L707-L709
